cwf/gateway/helm/cwf/templates/bin/_install_gateway.sh.tpl

```diff
@@ -62,9 +62,10 @@ mkdir -p "$INSTALL_DIR"
 MAGMA_GITHUB_URL="{{ .Values.cwf.repo.url }}"
 git -C "$INSTALL_DIR" clone "$MAGMA_GITHUB_URL" -b {{ .Values.cwf.repo.branch }}
 
-# TODO: Add this back once this code is included in a github version
-#TAG=$(git -C $INSTALL_DIR/magma tag | tail -1)
-#git -C $INSTALL_DIR/magma checkout "tags/$TAG"
+source /opt/magma/env/.env
+#if [ "$IMAGE_VERSION" != "latest" ]; then
```

Comment:
This section should be uncommented now. This will ensure that the version of `docker-compose.yml` is the same version as the images installed.